string username
int32 user_id
---
bool success
